config/main.py

```diff
@@ -1561,6 +1561,17 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
             click.echo("The config file {} doesn't exist".format(file))
             continue
 
+        if file_format == 'config_db':
+            file_input = read_json_file(file)
+
+            platform = file_input.get("DEVICE_METADATA", {}).\
+                get("localhost", {}).get("platform", {})
+            mac = file_input.get("DEVICE_METADATA", {}).\
+                get("localhost", {}).get("mac", {})
+
+            if not platform or not mac:
```

Comment:
>if [](http://example.com/codeflow?start=12&length=2)

For extreme corner case:
1. platform is None and mac is valid
2. platform is valid and mac is None
you can continue with load_sysinfo, please add some log_warning for future troubleshooting purpose.